DOCS.md

```diff
@@ -739,6 +756,18 @@ require('orgmode').setup({
 })
 ```
 
+### Edit Src
```

Comment:
Since we have a separate "section" for this, lets just add it up to the table of contents so it can be navigated.